### PR TITLE
Added prerequisites + Latch renewal period

### PR DESCRIPTION
**Pull request by RyuLindow**

### CODE REVIEW

File: Umbraco-Cloud/Set-Up/Umbraco-Latch/index.md [hunk 1]
```diff
@@ -6,6 +6,17 @@ versionFrom: 7.0.0
 
 All new projects on Umbraco Cloud are automatically protected by Umbraco Latch. This means, that the default Umbraco Cloud URL for your project as well as any new hostnames you add will be assigned a TLS certificate automatically.
 
+:::note
+Certificates last 3 months and are automatically renewed, as long as the prerequisites below are met.
+:::
+
+## Prerequisites
+In order for Latch to work and the certificate to be applied the root of the site that the domain is pointing to cannot have the status of 404 whether it's a custom 404 or one caused by an error.
```
Comment (sofietoft) [inline Umbraco-Cloud/Set-Up/Umbraco-Latch/index.md:14]: I'm not entirely sure I understand this sentence.
Is the point that, a domain cannot have a 404 status?

```suggestion
In order for Latch to work and the certificate to be applied to the root of the site that the domain is pointing to cannot have the status of 404 whether it's a custom 404 or one caused by an error.
```

File: Umbraco-Cloud/Set-Up/Umbraco-Latch/index.md [hunk 2]
```diff
@@ -6,6 +6,17 @@ versionFrom: 7.0.0
...
+
+Latch needs the site to have a status of 200 in order for the certificate to be applied and renewed. This means that every 3 months Latch will go through the renewal process and the prerequisites have to be met each time in order for it to work.
```
Comment (sofietoft) [inline Umbraco-Cloud/Set-Up/Umbraco-Latch/index.md:16]:
```suggestion
Latch needs the site to have a status of 200 in order for the certificate to be applied and renewed. This means that every 3 months Latch will go through the renewal process, and the prerequisites have to be met each time in order for it to work.
```

File: Umbraco-Cloud/Set-Up/Umbraco-Latch/index.md [hunk 3]
```diff
@@ -6,6 +6,17 @@ versionFrom: 7.0.0
...
+
+If you wish to have a custom 404 page on the root of your website then you will need to purchase and apply a custom certificate instead.
```
Comment (sofietoft) [inline Umbraco-Cloud/Set-Up/Umbraco-Latch/index.md:18]: Let's then add a link to the manual certificates article - what do you think?

```suggestion
If you wish to have a custom 404 page on the root of your website then you will need to purchase and apply a custom certificate instead.
```

**Review by sofietoft — CHANGES_REQUESTED** [review]
Thanks for updating this article @RyuLindow !

I've made a few suggestions.
Take a look, when you have the time 💪

File: Umbraco-Cloud/Set-Up/Umbraco-Latch/index.md (hunk 3 above)
Comment (RyuLindow) [reply Umbraco-Cloud/Set-Up/Umbraco-Latch/index.md:18]: Yeah, sounds like a good idea. Can you add it there? I'm unsure on how to do is partway through the PR 😐

Comment (sofietoft) [reply Umbraco-Cloud/Set-Up/Umbraco-Latch/index.md:18]: I believe the path is `../Manage-Hostnames/Security-Certificates/`.
Can add it I you like. Let me know!

**Review by sofietoft — COMMENTED** [review]
Thanks for applying some of the suggestions @RyuLindow ! Much appreciated.

About the link - I've suggested what I believe to be the correct path. Let me know if you're up for adding the link.

Also, line 14 - the sentence is a little too long, and I'm also not entirely sure I understand the meaning of the sentence. Would you be able to rewrite it - perhaps split it into two shorter sentences instead?

File: Umbraco-Cloud/Set-Up/Umbraco-Latch/index.md [hunk 4]
```diff
@@ -11,11 +11,13 @@ Certificates last 3 months and are automatically renewed, as long as the prerequ
 :::
 
 ## Prerequisites
-In order for Latch to work and the certificate to be applied to the root of the site that the domain is pointing to cannot have the status of 404 whether it's a custom 404 or one caused by an error.
+In order for Latch to work and the TLS certificate to be applied to the site, you need to fullfill some criteria. First of all, the content node that the domain is pointing to cannot have the status of 404 whether it's a custom 404 page or one caused by an error.
```
Comment (sofietoft) [inline Umbraco-Cloud/Set-Up/Umbraco-Latch/index.md:14]:
```suggestion
In order for Latch to work and the security certificate to be applied to the site, you need to fulfill some criteria. First of all, the content node that the domain is pointing to cannot have the status of 404 whether it's a custom 404 page or one caused by an error.
```

**Review by sofietoft — CHANGES_REQUESTED** [review]
Thanks for fixing the sentence @RyuLindow ! 😄 

Just a few suggestions;
Instead of using 'TLS', I've used 'security' instead, as it's a more general term for the certificate.

**Review by sofietoft — APPROVED** [review]
Thanks for applying the suggestions @RyuLindow !

I'll merge this in right away 🙌

**Pull request merged**